modules/swagger-codegen/src/main/resources/android/libraries/volley/apiInvoker.mustache

```diff
@@ -363,24 +363,32 @@ public class ApiInvoker {
   }
 
   public String invokeAPI(String host, String path, String method, List<Pair> queryParams, Object body, Map<String, String> headerParams, Map<String, String> formParams, String contentType, String[] authNames) throws ApiException, InterruptedException, ExecutionException, TimeoutException {
-    RequestFuture<String> future = RequestFuture.newFuture();
-    Request request = createRequest(host, path, method, queryParams, body, headerParams, formParams, contentType, authNames, future, future);
-    if(request != null) {
-       mRequestQueue.add(request);
-       return future.get(connectionTimeout, TimeUnit.SECONDS);
-    } else {
-      return "no data";
+    try {
+      RequestFuture<String> future = RequestFuture.newFuture();
+      Request request = createRequest(host, path, method, queryParams, body, headerParams, formParams, contentType, authNames, future, future);
+      if(request != null) {
+         mRequestQueue.add(request);
+         return future.get(connectionTimeout, TimeUnit.SECONDS);
+      } else {
+        return "no data";
+      }
+    } catch (UnsupportedEncodingException ex) {
+      throw new ApiException();
```

Comment:
@ecolleu What about `throw new ApiException(0, "UnsupportedEncodingException")` instead so the developers will have a clue about the exception?

Reply:
@wing328 You're right, this is better. I'm doing it. Thank you